Nodejs/Product/Nodejs/NodejsTools.vsct

```diff
@@ -334,6 +356,12 @@
       <IDSymbol name="AddNewFileGroup" value="0x1016"/>
     </GuidSymbol>
 
+    <GuidSymbol name="guidMigrateToJspsCmdSet" value="{5F602F6D-4592-448A-B321-2ECFA442BB29}">
+      <IDSymbol name="cmdidJspsProjectMigrate" value="0x206" />
+      <IDSymbol name="cmdidJspsProjectRevert" value="0x207" />
+      <IDSymbol name="cmdidJspsMigrate" value="0x208" />
```

Comment:
I think this is for a group rather than a command so I suggest renaming it (and update other instances of the old name in this file)
```suggestion
      <IDSymbol name="JspsMigrationGroup" value="0x208" />
```

Reply:
done in #2471